src/client/store/thunks/gameThunks.js

```diff
@@ -14,13 +14,26 @@ export const playAsGuestThunk = (name) => (dispatch) => {
 };
 
 export const createGameThunk = (rounds) => (dispatch) => {
-  console.log(rounds)
   return axios
     .post('/game/gameSession', {rounds})
-    .then(() => {
+    .then(({data}) => {
       dispatch(newGame(rounds));
+      axios.put('/user/session', {data})
     })
     .catch((e) => {
       console.log(e);
     });
 };
+
+export const findRandomGame = () => () => {
+  return axios
+    .get('/game/gameSession')
+      .then(({data}) => {
+        const {id} = data
+        axios.put(`/user/session`, {id})
```

Comment:
I believe the server currently works with params:
`// adds users to a game session
userRouter.put("/session/:id", async (req, res) => {
  try {
    const {id} = req.params;
    const { gameSessionId } = req.body;
    await Session.update({gameSessionId}, {where : {id}})
    res.sendStatus(200)
  } catch (e) {
    res.sendStatus(500);
    console.log(e);
  }
})`

Reply:
can you elaborate?

Reply:
You're sending id as a req.body to the session route, but put route for /user/session takes id as a parameter through user/session/:id